### ISSUE TITLE
dev-games/godot 3.0.2 stable ; 2.1.4 build failures

### ISSUE THREAD

**Issue by nado** [stefan-gr/abendbrot#109]
Hi,
Version 2.1.4 doesn't build anymore (gcc-7 not supported nor clang-6, all fixed in the last months it seems).
So it would be nice to be able to test the 3.0 branch.
Thanks for this ebuild ;)

Btw, according to this news (https://godotengine.org/article/dev-snapshot-godot-2-1-5-rc-1), 2.1.5 should also come soon and both 2.1 and 3.0 branch should get regular updates.

**Comment by stefan-gr** [comment]
Does merging the newest 9999 build work? I'm still not on gentoo but if a version bump is enough then I will do that for now.

**Comment by nado** [comment]
Yup, it works after copying 2.1.4 ebuild to 9999 which didn't have llvm slots yet.
Git master from godot is for the 3.0 branch, I'll try to provide ebuilds for latest stable releases from both branches.
I think I’ll slot them by the way, any comment on that?

Edit: while running repoman before proposing a PR, I discovered some tarball archives changed in godot repository, so I'm refetching all tarballs to verify their checksum is still valid.

**Comment by nado** [comment]
I also suggest get rid of older versions triggering warnings and errors because of the masked clang version dependency. This would mean rm everything <2.1.4.
